designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/RadioButton.kt

```diff
@@ -157,6 +160,46 @@ fun RadioButtonBlock(
     }
 }
 
+/**
+ * DHIS2 age field helper.
+ *
+ * @param orientation Controls how the radio buttons will be displayed, HORIZONTAL for rows or
+ * VERTICAL for columns.
+ * @param optionSelected controls which item is selected.
+ * @param onClick is a callback to notify which item has changed into the block.
+ */
+@Composable
+fun AgeFieldHelper(
```

Comment:
I think that I would extract this component and the enum to a single class called AgeFieldHelper

Reply:
done!